package.json

```diff
@@ -24,47 +23,44 @@
     "@material-ui/icons": "4.9.1",
     "@material-ui/lab": "^4.0.0-alpha.56",
     "axios": "0.19.0",
-    "json-server": "0.15.1",
-    "npm-run-all": "4.1.5",
     "prop-types": "^15.7.2",
     "react": "16.11.0",
```

Comment:
What about updating the version of react?

It's still set at:

```
    "react": "16.11.0",
    "react-dom": "16.11.0",
    "react-scripts": "^3.4.0",
```
CRA's latest build includes:

```
    "react": "^16.13.1",
    "react-dom": "^16.13.1",
    "react-scripts": "3.4.3"
```

Reply:
Thanks @tgrrr - my goal with this PR is to get us up to the safest, stable version of our packages. There were a few cases where we were a good couple of major versions behind. I didn't opt in to upgrade us to the latest versions of packages because in my experience, sometimes latest versions of packages can get us in dependency hell. For example, some months ago, Testing Library was updated but it also required Jest to be at a certain version. As we were using Jest from CRA, we were pinned at their version. In the end, I ended up having to set up Jest manually for this project in order for our testing infrastructure to work cohesively. This is an extreme case, but I'd like to not have to deal with those kinds of problems too often. 

As the versions you mentioned are only ~2 minor / patch versions behind, I think we are safe with where we are here.

Reply:
I also had a jest conflict with the current config @angelocordon

Reply:
16.3 is a significant update to react from 16.11 https://github.com/facebook/react/releases

Reply:
@tgrrr re: Jest conflicts - can you expand more on what that conflict is on your end? I'm not seeing any issues on my end other than an async syntax error which I just fixed. Other than that, running `npm run test` passes all suites 👍🏽. But if you're having something bigger than that, it may be worth filing an issue so we can resolve it outside of this. 

re: React updates - the changes from 16.11 to 16.13 don't apply to us, we're not using lazy or memo, nor do we have a multi-root app. We're not dealing with concurrent mode, nor do I see any of the issues from their updates with ReactDOM applying to us. I think we'll eventually get to 16.13 in due time, but as of now, this feels out of the scope of this PR.